eventauth_test.go

```diff
@@ -393,17 +413,6 @@ func TestAllowedFirstJoin(t *testing.T) {
 		}, {
 			"type": "m.room.member",
 			"state_key": "@u1:a",
-			"sender": "@u1:a",
-			"room_id": "!r1:a",
-			"event_id": "$e4:a",
-			"content": {"membership": "join"},
-			"prev_events": [[0, {}]],
-			"unsigned": {
-				"not_allowed": "The prev_events event ID is not a string"
-			}
```

Comment:
I can't actually use this test case anymore because the prev_events are parsed inside the constructor for events rather than inside the auth checks.